docs/wallets/index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Use Ethereum
+meta:
+  - name: description
+    content: The basic information you need to get started using Ethereum.
+  - property: og:title
+    content: Use Ethereum
+  - property: og:description
+    content: The basic information you need to get started using Ethereum.
+lang: en-US
+---
+
+# Ethereum Wallets {#wallets}
+
+<div class="featured">
+
+  **Want to learn more about Ethereum Wallets?** This page will explain the basics and provide you with a few simple options. 
+
+</div>
+
+## What is an Ethereum wallet, and which one should I use?
+
+Wallets are applications that make it easy to hold and send [ETH](https://www.ethereum.org/eth), as well as interact with [applications built on Ethereum](https://www.ethereum.org/use).
```

Comment:
`[ETH](/eth)` & [applications built on Ethereum](/dapps)

Love to see the interlinking tho! 😄